src/pkg/services/m365/api/graph/service.go

```diff
@@ -416,7 +420,9 @@ func (aw *adapterWrap) Send(
 		err = stackWithCoreErr(ictx, err, 1)
 		e = err
 
-		if IsErrConnectionReset(err) || connectionEnded.Compare(err.Error()) {
+		if IsErrConnectionReset(err) ||
+			connectionEnded.Compare(err.Error()) ||
```

Comment:
hmm, graph/errors.go should have a pattern to construct these tests internally.  Can you add a todo that we need to migrate them there?  it feels a little hacky with the current implementation.